web_src/js/features/repo-migrate.js

```diff
@@ -31,6 +33,7 @@ export function initRepoMigrationStatusChecker() {
     if (data.status === 3) {
       hideElem('#repo_migrating_progress');
       hideElem('#repo_migrating');
+      showElem('#repo_migrating_retry');
```

Comment:
@delvh The retry button is displayed only if it fails

Reply:
Can you add final screenshots one or two will be more than enough..

Reply:
![7UJDNM671RI})EA8~~XPL39](https://github.com/go-gitea/gitea/assets/3371163/e088fd63-5dcc-4bc6-8849-7db3086511b7)
![T83F1WL9)VGHR@MB956$VT9](https://github.com/go-gitea/gitea/assets/3371163/744425bb-dde1-4315-be2e-5c99ac3a44d4)